android/local.properties

```diff
@@ -0,0 +1 @@
+flutter.sdk=/Users/diogosilva/Development/flutter
```

Comment:
Can we remove this file or add it to `gitignore` since it is using path on your personal laptop?